examples/ola-streaming-client.cpp

```diff
@@ -78,11 +84,30 @@ int main(int argc, char *argv[]) {
 
   if (FLAGS_dmx.str().empty()) {
     string input;
+    bool have_universe = false;
+    unsigned int read_universe = FLAGS_universe;
+
     while (!terminate && std::cin >> input) {
-      ola::StringTrim(&input);
-      SendDataFromString(&ola_client, FLAGS_universe, input);
+      if (!have_universe && FLAGS_universe_from_source) {
+        if (!ola::StringToInt(input, &read_universe, true)) {
+          OLA_FATAL << "Could not convert universe number.";
```

Comment:
It would be nice to put the string it's tried to process in the message to aid debugging.